covid_xprize/oxford_data/oxford_data.py

```diff
@@ -348,8 +348,18 @@ def convert_smooth_cases_per_100K_to_new_cases(smooth_cases_per_100K,
                                                window_size,
                                                prev_new_cases_list,
                                                pop_size):
-    return (((window_size * pop_size) / 100000.) * smooth_cases_per_100K \
-            - np.sum(prev_new_cases_list[-(window_size-1):])).clip(min=0.0)
+    # Smooth cases per 100K -> Smooth cases
```

Comment:
This function could use a docstring. It's pretty hard to understand what it does from the code...
I'm glad you have a unit test for it because I can't easily tell from the code if the logic is correct or off by 1...